modules/javafx.graphics/src/main/java/javafx/application/ConditionalFeature.java

```diff
@@ -150,6 +150,15 @@ public enum ConditionalFeature {
      */
     UNIFIED_WINDOW,
 
+    /**
+     * Indicates that a system supports {@link javafx.stage.StageStyle#EXTENDED}.
+     * <p>
+     * This feature is currently supported on Windows, Linux, and macOS.
```

Comment:
would it make sense to add more information similarly to UNIFIED_WINDOW on L151?
maybe pointing to different levels of support?

I don't know whether it makes sense to bring the whole table from the JEP here, but maybe somewhere (HeaderBar)?

Reply:
It links directly to `StageStyle.EXTENDED`, which contains lots of information. I don't think we need it here.

By the way, there are no support levels. The feature is supported in its entirety, as specified, on all desktop platforms. The table in the JEP is just informational to help OpenJFX developers understand what we're talking about. It's not a specification, and it can't be a specification because we either don't control things (for example, we have no influence on rounded corners, this could be changed with any future OS version) or because we'd be specifying us into a corner. There's no point prescribing every minute detail, and then being responsible to keep it working _exactly_ as described, even though the semantics are unchanged.

The relevant parts are specified in `StageStyle.EXTENDED` and `HeaderBar`, and I wouldn't go further than this. We don't specify the details of `StageStyle.DECORATED` for each OS; why should we do it for `StageStyle.EXTENDED`?

Reply:
I don't want to do this, not because I don't want to write documentation, but because I see a very real risk of over-specifying these kinds of things. For app developers, it shouldn't matter if a close button is drawn by the OS or by JavaFX, the only thing that matters is that they _are_ drawn, and it is not the responsibility of the application developer to do it.

The specification of `StageStyle.EXTENDED` says:
```
* An extended window has the default window buttons (minimize, maximize, close), but no system-provided
* draggable header bar.
```

That's what is important in my opinion: if you choose this stage style, you will get window buttons (however this is done is of no interest), but you will not get a header bar.